cmd/mr_test.go

```diff
@@ -5,25 +5,64 @@ import (
 	"io/ioutil"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 	"strings"
 	"testing"
 
 	"github.com/acarl005/stripansi"
 	"github.com/stretchr/testify/require"
 )
 
+func cleanupMR(t *testing.T, targetRepo string, cmdDir string, MRtitle string) {
+	openMRcmd := exec.Command("sh", "-c", fmt.Sprintf("%s mr list %s | grep -m1 \"%s\" | cut -c2- | awk '{print $1}'", labBinaryPath, targetRepo, MRtitle))
+	openMRcmd.Dir = cmdDir
+	b, err := openMRcmd.CombinedOutput()
+	if err != nil {
+		t.Log(string(b))
+		t.Fatal(err)
+	}
+
+	openMR, err := strconv.Atoi(strings.TrimSpace(string(b)))
+	if err != nil {
+		t.Log(string(b))
+		return
+	}
+
+	if openMR <= 0 {
+		// no open MRs
+		return
+	}
+
+	// close the existing MR
+	closeMRcmd := exec.Command("sh", "-c", fmt.Sprintf("%s mr close %d\n", labBinaryPath, openMR))
+	closeMRcmd.Dir = cmdDir
+	c, err := closeMRcmd.CombinedOutput()
+	if err != nil {
+		t.Log(string(c))
+		t.Fatal(err)
+	}
+}
+
+func deleteMR(t *testing.T, targetRepo string, cmdDir string, mrID string) {
```

Comment:
I think it would be better if you change `deleteMR` to `closeMR`... I'm implementing the `lab mr delete` command right now and their meaning are quite different. Today we have the `delete` as an alias for `close`, which is really misleading.

Reply:
Fixed, and I also renamed the "delete" test functions to "close".